### DISCUSSION TITLE
OBC Race stats show gap and driver ahead and behind

### DISCUSSION THREAD

**Discussion by JJWatMyself** [f1multiviewer/issue-tracker#283] [Feature requests]
Imagine if the OBC live timing showed the driver ahead and behind...

![unknown 1](https://user-images.githubusercontent.com/22590445/201198347-c4214046-ef9b-4485-b893-fa1554c21b95.png)

**Comment by Jurtaani** [comment]
This has been on my Christmas gift wishlist since forever, but i don't have the means to make such clear example pictures as you.
This would make it so much easier to keep an eye on specific driver and see if they have been keeping up with the phase or if drivers from behind have been catching up with them.

**Reply by JJWatMyself** [reply]
Something I didn't think of until now is their DRS status also.

**Comment by JJWatMyself** [comment]
Ok, so other things for the OBC live stat overlay for driver ahead and behind

DRS
Tire and laps
Blue flag driver ahead
Gap to blue driver ahead
Blue flag driver logo info, e.g. LAT  <-- maybe hardcode this at first as will be accurate most of the time...

One last idea because we're talking about blue flags...

Why stop there, just add current sector flag status, based on where driver is <--- I've been very patient... we have smoe awesome 64x64 GIF files already created.
